stl/inc/__msvc_all_public_headers.hpp

```diff
@@ -139,13 +139,11 @@
 #include <vector>
 
 #ifndef _M_CEE_PURE
+#include <__msvc_cxx_stdatomic.hpp>

```

Comment:
This isn't a public header, per se. Should we TRANSITION and switch back to including `stdatomic.h` once things stabilize? (Also in `header-units.json` and `P0943R6_stdatomic_h``.)

Reply:
well, header-units lists many non-public headers. stdatomic_h 🤷 We could transition it back, but it's reasonable to just include the one that's part of "us" so to speak